codes/classical/bits/Storage Codes.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+code_id: storage 
+physical: bits
+logical: bits
+
+name: 'recoverable distributed storage system code'
+introduced: '\cite{https://arxiv.org/abs/1408.4862}'
+
+
+description: 'It is a code for a distributed storage system such that every node can be recovered from its neighbors.The the nodes in the distributed storage system may not be connected to every other node in the system. '
```

Comment:
"neighbors" in the first sentence undefined. The second sentence is not important here. Replace with:
A storage code is a way of representing information in a distributed storage system wherein the contents of the file is spread across multiple nodes. In the event of node failure the contents of the node can be recovered from a subset of other nodes in the system, called the helper nodes.